gpkit/interactive/ractor.py

```diff
@@ -106,7 +119,8 @@ def ractorjs(title, model, update_py, ranges, constraint_js=""):
                   varname = 'var'+i
                   idx = $w.ractive.data[varname]
                   console.log("$w-"+varname);
-                  document.getElementById("$w-"+varname).innerText = Math.round(100*$w.ranges[varname][idx])/100
+                  document.getElementById("$w-"+varname).innerText
```

Comment:
Not for now, I don't think. Maybe when we have a full web framework that should become a goal. I fixed it on master.